tests/helpers/create-test-function.js

```diff
@@ -0,0 +1,9 @@
+import { test } from 'qunit';
+
+export default function(fn) {
+    return function(given, expected, description) {
+        test(description, function (assert) {
+            assert.deepEqual(fn(given), expected);
+        })
+    }
+}
```

Comment:
trailing newline

Reply:
https://github.com/emberjs/ember-string/blob/master/.editorconfig#L12 ??

Reply:
Now I see is not being consistent. WTHeck? Fixing it.